services/direct_delivery/javascript_validation.js

```diff
@@ -0,0 +1,42 @@
+const request = require('request');
+
+const mailsacKey = 'YOUR_API_KEY_HERE';
+
+const options = {
+    url: 'https://mailsac.com/api/addresses/mycustomer@gmail.com/messages',
+    headers: {
+        'Mailsac-Key': mailsacKey
+    }
+};
+
+function callback(error, response, body) {
+    if (!error && response.statusCode == 200) {
+        const info = JSON.parse(body);
+        console.log(info);
+    }
+}
+
+request(options, callback);
```

Comment:
Even though this may be better coding practice, it is often better to write example code samples in procedural order.

Also, another example use superagent instead of request. While both are popular - request probably moreso, honestly - superagent is more readable for example code, IMO.

```javascript
const superagent = require('superagent')

superagent
  .get('https://mailsac.com/api/addresses/mycustomer@gmail.com/messages')
  .set('Mailsac-Key', 'YOUR_API_KEY_HERE')
  .then()
  .catch()
```